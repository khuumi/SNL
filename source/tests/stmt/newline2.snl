do func1 to a, 7, do func2
