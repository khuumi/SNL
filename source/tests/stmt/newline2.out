Expr (Call func1 [(Id Global a), (Int 7), (Call func2 [])])